### ISSUE TITLE
Latest crashes in Bitwig

### ISSUE THREAD

**Issue by TjShredder** [michaelwillis/dragonfly-reverb#89]
Version 3.2.3 is unusable it crashes in Bitwig immediately. Not only the host, the audio engine as well. I have never seen that before...
[BitwigPluginHost_2020-12-09-232902_Tjs-Pro.crash.zip](https://github.com/michaelwillis/dragonfly-reverb/files/5669153/BitwigPluginHost_2020-12-09-232902_Tjs-Pro.crash.zip)

**Comment by michaelwillis** [comment]
@TjShredder Do all four of the 3.2.3 Dragonfly plugins crash bitwig? Which version were you using before you upgraded? Are you loading a Bitwig session that was saved with a prior version of Dragonfly Reverb? Does Bitwig crash if you create a new session and then add one of the Dragonfly plugins?

**Comment by TjShredder** [comment]
Yes, they do. Also on my other machine. Both run Mojave... Freshly loaded into a new project...
I guess it was the stable version before. I put them back from my time machine, but where do I see what version it is? (they work fine...)

**Comment by michaelwillis** [comment]
@TjShredder Click on the question mark, it will show version number and credits.

**Comment by TjShredder** [comment]
Ok, the working version is 3.0.0

**Comment by michaelwillis** [comment]
@TjShredder I know this bug report is really old, but I'm getting back to working on this project after not putting much time into it last year. Please let me know if you still experience this issue in the new [release 3.2.6](https://github.com/michaelwillis/dragonfly-reverb/releases/tag/3.2.6).

**Comment by TjShredder** [comment]
I had no problems since a long time, but didn't use it a lot either. A quick test reveals no problem in Bitwig 4.2.2...
